README.md

```diff
@@ -1 +1,184 @@
-# DeepStack-CameraUI
\ No newline at end of file
+# DeepStack-CameraUI
+DeepStack-CameraUI is a real-time computer vision web-application. Powered by Deepstack and Streamlit, this application performs realtime AI processing such as **object detection**, **face recognition**, **face detection** and **custom detection** on video frames from WebCam, Wired Camera or IP camera through an interactive web interface.
+
+# Installation
+
+To run DeepStack-CameraUI, you need to install the following:
+* **Python 3.8** 
+
+  - [Windows](https://www.python.org/ftp/python/3.8.0/python-3.8.0-amd64.exe) 
+  - [MacOS](https://www.python.org/ftp/python/3.8.0/python-3.8.0-macosx10.9.pkg)
+  - [Ubuntu](https://www.python.org/ftp/python/3.8.0/Python-3.8.0.tar.xz)
+
+* [Deepstack](https://docs.deepstack.cc/object-detection/index.html) 
+* Clone the reporsitory 
+* Install the dependencies by navigating into the project repository with `pip install -r requiremnets.txt`
+
+# Run DeepStack-CameraUI
+To run this streamlit application **locally** , follow these steps 
+
+* Step 1 : Run deepstack and streamlit
+
+Execute the following command in the app folder, first run Deepstack with this command:
+```
+docker run -e VISION-FACE=True -v localstorage:/datastore -p 80:5000 deepquestai/deepstack
+```
+Check [Deepstack Documantation](https://docs.deepstack.cc/) ,then navigate to the src folder and run the streamlit app using :
+```
+streamlit run app.py
+```
+
+* step 2 : Load the application and configure the settings
+
+  - Select your API
+  - Insert deepstack URL
+  - Insert your camera address (0 for Webcam, -1 on Ubuntu or IP camera address)
+  - Press on **Start** button
+
+Now a real-time detection deom your camera feed would be displayed in the main app interface.
```

Comment:
Let this be 

Now you will see real-time detection and video feed from your camera displayed in the main app interface.